Header header
uint8 motor_id
float32 rpm
float32 bus_voltage
float32 bus_current
float32 temperature
uint32 fault_flag
